src/lib/finishables.ts

```diff
@@ -161,6 +164,17 @@ export const finishables: Finishable[] = [
 		aliases: Naxxus.aliases,
 		kill: ({ accumulatedLoot }) => rollNaxxusLoot(1, accumulatedLoot)
 	},
+	{
+		name: 'Nex',
+		cl: nexCL.filter(i => i !== itemID('Frozen key')),
+		kill: () => {
+			const loot = new Bank(NexMonster.table.kill(1, {}));
+			if (roll(82)) {
+				loot.add(randArrItem(nexUniqueDrops), 1);
+			}
+			return loot;
```

Comment:
This `roll` here is using a magic number, that needs to be abstracted & shared properly with the real nex code. e.g. `export const NEX_UNIQUE_DROPRATE = (teamSize: number) => something * teamSize;`